tradelogs/cmd/kyc-reaggregate/main.go

```
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"log"
+	"os"
+	"time"
+
+	libapp "github.com/KyberNetwork/reserve-stats/lib/app"
+	"github.com/KyberNetwork/reserve-stats/lib/cq"
+	"github.com/KyberNetwork/reserve-stats/lib/influxdb"
+	"github.com/KyberNetwork/reserve-stats/lib/timeutil"
+	"github.com/KyberNetwork/reserve-stats/tradelogs/common"
+	tradelogcq "github.com/KyberNetwork/reserve-stats/tradelogs/storage/cq"
+	"github.com/KyberNetwork/reserve-stats/tradelogs/workers"
+	"github.com/urfave/cli"
+	"go.uber.org/zap"
+)
+
+const (
+	defaultDB    = "reserve_stats"
+	fromTimeFlag = "fromTime"
+	toTimeFlag   = "toTime"
+	//This is 01 Feb 2018 UTC
+	defaultFromTime = 1517443200000
+)
+
+func main() {
+	app := libapp.NewApp()
+	app.Name = "Trade Logs KYC re-aggregate"
+	app.Version = "0.0.1"
+	app.Action = run
+	app.Flags = append(app.Flags,
+		cli.Uint64Flag{
+			Name:   fromTimeFlag,
+			Usage:  "re-KYC from time. Default is 00:00:00",
```

I think the default time is in usage is misleading.

---

Also could the usage state something about format if from/to time (in millis/micro/nano)?